kubernetes/README.md

```diff
@@ -1,184 +1,137 @@
-# Deploying the 1Password SCIM bridge on Kubernetes
+# Deploy 1Password SCIM Bridge on Kubernetes
 
-This example explains how to deploy the 1Password SCIM bridge on a Kubernetes cluster. It assumes your Kubernetes cluster supports "load balancer" services.
+*Learn how to deploy 1Password SCIM Bridge using Kubernetes and load balancer services.*
 
-You can modify this deployment to suit your environment and needs.
-
-If you are deploying to the Azure Kubernetes Service, you can refer to our [detailed deployment guide instead](https://support.1password.com/cs/scim-deploy-azure/).
-
-The deployment process consists of these steps:
-
-1. Create a Kubernetes Secret with your `scimsession` file
-2. (Optional) Create Kubernetes Secrets for Google Workspace
-3. Deploy 1Password SCIM bridge to your cluster
-4. Create a DNS record for SCIM bridge
-5. Configure Let's Encrypt
-
-## Structure
-
-- [`op-scim-deployment.yaml`](./op-scim-deployment.yaml): The Deployment object for the SCIM bridge container.
-- [`op-scim-service.yaml`](./op-scim-service.yaml): Public load balancer for SCIM bridge to enable connectivity for your idenitty provider.
-- [`op-scim-config.yaml`](./op-scim-config.yaml): Configuration for the SCIM bridge Deployment.
-- [`redis-deployment.yaml`](./redis-deployment.yaml): A Redis cache deployed in the cluster.
-- [`redis-service.yaml`](./op-scim-service.yaml): Kubernetes Service for the Redis cache to enable connectivity inside the cluster.
-- [`redis-config.yaml`](./redis-config.yaml): Configuration for the Redis cache.
-
-## Preparing
-
-Ensure you've read through the [PREPARATION.md](/PREPARATION.md) document before beginning deployment.
+> **Note**
+>
+> If you use Azure Kubernetes Service, learn how to [deploy the SCIM bridge there](https://support.1password.com/scim-deploy-azure/).
 
-## Clone `scim-examples`
+**Table of contents:**
 
-As seen in [PREPARATION.md](/PREPARATION.md), you’ll need to clone this repository using `git` into a directory of your choice.
+- [Before you begin](#before-you-begin)
+- [Step 1: Create the `scimsession` Kubenetes Secret](#step-1-create-the-scimsession-kubernetes-secret)
+- [Step 2: Deploy 1Password SCIM Bridge to the Kubernetes cluster](#step-2-deploy-1password-scim-bridge-to-the-kubernetes-cluster)
+- [Step 3: Create a DNS record](#step-3-create-a-dns-record)
+- [Step 4: Configure Let's Encrypt](#step-4-configure-lets-encrypt)
+- [Step 5: Test the SCIM bridge](#step-5-test-the-scim-bridge)
+- [Step 6: Connect your identity provider](#step-6-connect-your-identity-provider)
+- [Update your SCIM Bridge](#update-your-scim-bridge)
+- [Appendix: Resource recommendations](#appendix-resource-recommendations)
+- [Appendix: Customize your deployment](#appendix-customize-your-deployment)
 
-```bash
-git clone https://github.com/1Password/scim-examples.git
-```
+## Before you begin
 
-You can then browse to the Kubernetes directory:
+Before you begin, read the [PREPARATION.md](/PREPARATION.md) document.
```

Comment:
Use the name of the doc rather than the filename for consistency and readability:

```suggestion
Before you begin, read the [Preparation Guide](/PREPARATION.md) document at the root of this repository.
```